src/main/java/org/folio/rest/impl/UploadBatchVoucherExportHelper.java

```diff
@@ -72,19 +74,19 @@ public Future<Void> uploadBatchVoucherExport(BatchVoucherExport batchVoucherExpo
 
   public Future<Void> uploadBatchVoucher(BatchVoucherUploadHolder uploadHolder) {
     try {
-      FtpUploadService ftpUploadService = new FtpUploadService(ctx, uploadHolder.getExportConfig().getUploadURI());
-      return ftpUploadService.login(uploadHolder.getCredentials().getUsername(), uploadHolder.getCredentials().getPassword())
-        .onSuccess(log::info)
-        .compose(v -> {
-          String fileName = generateFileName(uploadHolder.getBatchVoucher(), uploadHolder.getFileFormat());
-          BatchVoucher batchVoucher = uploadHolder.getBatchVoucher();
-          String format = uploadHolder.getExportConfig().getFormat().value();
-          String content = batchVoucherService.convertBatchVoucher(batchVoucher, format);
-
-          return ftpUploadService.upload(ctx, fileName, content);
-        })
-        .mapEmpty();
-    } catch (URISyntaxException e) {
+      String fileName = generateFileName(uploadHolder.getBatchVoucher(), uploadHolder.getFileFormat());
+      BatchVoucher batchVoucher = uploadHolder.getBatchVoucher();
+      String format = uploadHolder.getExportConfig().getFormat().value();
+      String content = batchVoucherService.convertBatchVoucher(batchVoucher, format);
+
+      if (uploadHolder.getExportConfig().getFtpFormat().equals(ExportConfig.FtpFormat.FTP)) {
```

Comment:
```suggestion
      if (ExportConfig.FtpFormat.FTP.equalsuploadHolder.getExportConfig().getFtpFormat()))) {
```
It will allow NPE in case when ftpFormat is null , also you can use == to compare enums